data_model/src/lib.rs

```diff
@@ -192,18 +192,16 @@ impl std::error::Error for ParseError {}
 /// Validation of the data model entity failed.
 #[derive(Debug, Display, Clone)]
 pub struct ValidationError {
```

Comment:
No, this is because @Erigara [suggested](https://github.com/hyperledger/iroha/pull/2694#discussion_r962767736) it.

And indeed we often use `&'static str` for this kind of error. What's wrong with this approach?